app/src/main/java/com/smlnskgmail/jaman/hashchecker/components/hashcalculator/api/HashType.java

```diff
@@ -20,7 +20,14 @@ public enum HashType implements ListItem {
     SHA3_384(Constants.SHA3_384, "sha3_384"),
     SHA3_512(Constants.SHA3_512, "sha3_512"),
     CRC_32(com.smlnskgmail.jaman.hashchecker.components.hashcalculator.hash.Constants.CRC_32, "crc32"),
```

Comment:
Can you add constants without write full package name?

Reply:
Rename Contants to CustomConstant